packages/react-dom-bindings/src/client/ReactDOMComponent.js

```diff
@@ -273,6 +274,7 @@ function setProp(
   key: string,
   value: mixed,
   props: any,
+  prevValue: mixed,
 ): void {
   switch (key) {
     case 'children': {
```

Comment:
Move the validateDOMNesting single text child logic here? I don't think it needs the ancestors for that check, just the parent tag name.

Reply:
I'll do that separately. Seems like I should break it out of the validateDOMNesting helper into separate one that doesn't require the full ancestor info.

Reply:
yea

Reply:
https://github.com/facebook/react/pull/26594